core/sources/config.move

```diff
@@ -95,10 +95,16 @@ module aptos_names::config {
         100000000
     }
 
+    // !!!!! DEPRECATED !!!!! please use is_enabled_for_nonadmin()
     public fun is_enabled(): bool acquires ConfigurationV1 {
         read_bool_v1(@aptos_names, &config_key_enabled())
     }
 
+    // Returns true if enabled flag is true or signer is admin
+    public fun is_enabled_for_nonadmin(sign: &signer): bool acquires ConfigurationV1 {
```

Comment:
nit: Why does this need to take a signer if it doesn't need to do anything with it? Why not take an address instead?

Reply:
This is kinda a view function anyway that has no side effect. The callers of this function can enforce that signer themselves.